lib/rex/proto/ldap/server.rb

```diff
@@ -228,25 +285,50 @@ def encode_ldap_response(msgid, code, dn, msg, tag)
         # @param attrflt [Array, Symbol] LDAP attribute filter
         #
         # @return [Array] Query matches
-        def search_ldif(filter, msgid, attrflt = :all)
-          return [] if @ldif.nil? || @ldif.empty?
-
-          ldif.map do |dn, entry|
-            next unless filter.match(entry)
 
+        def search_result(filter, msgid, attrflt = :all)
```

Comment:
Update yard docs above to note the change in behavior here that causes this methods name change. 

```
        # Search provided ldif data for query information if no `ldif` was provided a random search result will be generated.
```